lib/load_admin_info.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'yaml'
+
+module BCDiceAPI
+  def self.load_admin_info
+    config = {}
```

Comment:
```suggestion
    config = {
      name: '',
      url: '',
      email: '',
    }
```

ここで初期化しておかないと、何も設定してない時に表示されません